src/asmcnc/skavaUI/screen_go.py

```diff
@@ -649,7 +649,11 @@ def start_or_pause_button_press(self):
 
         log('start/pause button pressed')
         if self.is_job_started_already:
-            self._pause_job()
+            if not self.m.is_machine_paused:
+                self._pause_job()
```

Comment:
probably better to put the button image change here instead of in the pause screen, makes it more general and keeps it local to just the go screen